packages/cactus-plugin-satp-hermes/src/test/typescript/unit/services.test.ts

```diff
@@ -153,15 +153,16 @@ describe("SATP Services Testing", () => {
       "hex",
     );
     sessionData.serverGatewayPubkey = sessionData.clientGatewayPubkey;
-    sessionData.originatorPubkey = sessionData.clientGatewayPubkey;
-    sessionData.beneficiaryPubkey = sessionData.clientGatewayPubkey;
+    sessionData.originatorPubkey = "MOCK_ORIGINATOR_PUBKEY";
+    sessionData.beneficiaryPubkey = "MOCK_BENEFICIARY_PUBKEY";
     sessionData.digitalAssetId = "MOCK_DIGITAL_ASSET_ID";
     sessionData.assetProfileId = "MOCK_ASSET_PROFILE_ID";
     sessionData.verifiedOriginatorEntityId =
       "MOCK_VERIFIED_ORIGINATOR_ENTITY_ID";
     sessionData.verifiedBeneficiaryEntityId =
       "MOCK_VERIFIED_BENEFICIARY_ENTITY_ID";
     sessionData.receiverGatewayOwnerId = "MOCK_RECEIVER_GATEWAY_OWNER_ID";
```

Comment:
Likewise for other mocked attributes

Reply:
This is not applicable here, in the test file